### ISSUE TITLE
Wrap UnionTypeNode

### ISSUE THREAD

**Issue by kjpou1** [dsherret/ts-simple-ast#241]
Would open up navigation without resorting to ts.UnionTypeNode <=> ts-simple-ast casting.

- [ ] getTypeNodes
etc..

Just a suggestion could be a couple of helper methods like `is` or `has` a `NullKeyword` and `UndefinedKeyword`.  

example with casting:

```js
function foo (node : sast.Node)
{
    const unionTypeNode = <ts.UnionTypeNode>node.compilerNode;
    for (let i = 0, n = unionTypeNode.types.length; i < n; i++) {
      const type = unionTypeNode.types[i];
      console.log((<any>ts).SyntaxKind[type.kind]);
    }
}
```

**Comment by dsherret** [comment]
Hi @kjpou1, this is implemented in 6.10.0.

Also, I added `TypeGuards.isUndefinedKeyword`. The `TypeGuards.isNullLiteral` has been around for a bit.

Let me know if you find any issues with it.

**Comment by kjpou1** [comment]
Works like a charm.  Thanks @dsherret !
